include/valijson/utils/nlohmann_json_utils.hpp

```diff
@@ -19,13 +20,17 @@ inline bool loadDocument(const std::string &path, nlohmann::json &document)
     }
 
     // Parse schema
+#if VALIJSON_USE_EXCEPTION
     try {
         document = nlohmann::json::parse(file);
     } catch (std::invalid_argument const& exception) {
         std::cerr << "nlohmann::json failed to parse the document\n"
             << "Parse error:" << exception.what() << "\n";
         return false;
     }
+#else
+    document = nlohmann::json::parse(file, nullptr, false);
+#endif
```

Comment:
Could be worth checking whether the result is discarded here, and returning false if so.

Reply:
Great idea, I'll add it.